.travis.yml

```diff
@@ -1,12 +1,10 @@
 language: ruby
 
-jdk: openjdk8
-
 cache:
   - bundler
 
 after_success:
-  - test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "master" && bundle exec rake publish
+  - test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "master" && bundle exec rake
```

Comment:
@sbastn can you explain the reason you kept this config?

Reply:
I am going to remove the travis config since everything is done through Netlify at this point.